.github/workflows/docker.yaml

```diff
@@ -72,7 +74,6 @@ jobs:
       - name: Publish to GitHub Packages
         uses: docker/build-push-action@v4
         with:
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64 # ,linux/arm64 (removed for debugging)
```

Comment:
Let's not forget to unremove as many people use ARM including myself. : )

Reply:
done